mypy/modulefinder.py

```diff
@@ -449,10 +454,8 @@ def _is_compatible_stub_package(self, stub_dir: str) -> bool:
         """
         metadata_fnam = os.path.join(stub_dir, 'METADATA.toml')
         if os.path.isfile(metadata_fnam):
-            # Delay import for a possible minor performance win.
```

Comment:
Fine to remove it since configparser.py already unconditionally imports tomllib anyway.